client/src/pages/GuidePage/GuidePage.component.tsx

```diff
@@ -82,6 +83,10 @@ export function GuidePage(props: GuidePageProps): JSX.Element {
         <MenuButton start='close' content='text' end='check_circle' size='xl' loading/>
       </Block>
 
+      <Block gap='xs' direction='xy' align='center' padding='xl'>
+        <Format v='amount' value={123.12}/>
```

Comment:
Add more examples with long and short digits. It allows you to see formats in ui, but yeah test implementation is better